PhysicsTools/HeppyCore/python/framework/config.py

```diff
@@ -97,6 +104,12 @@ def build_name(self):
         name = '_'.join([class_name, self.instance_label])
         return name 
 
+    def clone(self, **kwargs):
```

Comment:
- Why not reimplementing __copy__ and / or __deepcopy__ ? the meaning of clone is not standard in python.
- Please provide a docstring. 
- Why the kwargs pararametr?
- Do not import modules in the code but at the top of the module